api/src/services/sheets/sheets.js

```diff
@@ -0,0 +1,56 @@
+import { requireAuth } from 'src/lib/auth'
+import {
+  updateRestaurant,
+  restaurantByOwnerId,
+} from '../restaurants/restaurants'
+
+import { google } from 'googleapis'
+import fs from 'fs'
+
+export const createSheet = async () => {
+  requireAuth()
+
+  const auth = new google.auth.GoogleAuth({
+    credentials: {
+      client_email: process.env.CLIENT_EMAIL,
+      private_key: process.env.PRIVATE_KEY.replace(/\\n/gm, '\n'),
+    },
+    scopes: ['https://www.googleapis.com/auth/drive'],
+  })
+
+  const authClient = await auth.getClient()
+
+  const driveClient = google.drive({
+    version: 'v3',
+    auth: authClient,
+  })
```

Comment:
Can we add a new line here?

```suggestion
  })
  
```